airflow/models.py

```diff
@@ -2010,6 +2129,105 @@ def get(cls, key, session, deserialize_json=False):
         return v
 
 
+class XCom(Base):
+    """
+    Base class for XCom objects.
+    """
+    __tablename__ = "xcom"
+
+    id = Column(Integer, primary_key=True)
+    key = Column(String)
+    value = Column(PickleType(pickler=dill))
+    timestamp = Column(DateTime, server_default=func.current_timestamp())
+    visible_on = Column(DateTime, nullable=False)
```

Comment:
could also be called `execution_date` for consistency with the TaskInstance pk, but it may be confusing too. I'm ok with `visible_on`.